networkit/cpp/distance/EffectiveDiameter.cpp

```diff
@@ -32,7 +32,7 @@ void EffectiveDiameter::run() {
     // the current distance of the neighborhoods
     count h = 1;
     // number of nodes that need to be connected with all other nodes
-    auto threshold = static_cast<count>(std::ceil(ratio * G->numberOfNodes()) + 0.5);
+    auto threshold = static_cast<count>(std::round(std::ceil(ratio * G->numberOfNodes())));
```

Comment:
Uhm, `std::ceil` returns an integer anyway, right? So what's the point of the `std::round`?

Reply:
That's true, I removed it.